src/Microsoft.AspNetCore.Blazor/Components/ParameterCollectionExtensions.cs

```diff
@@ -66,6 +66,10 @@ public static class ParameterCollectionExtensions
             }
         }
 
+        internal static IEnumerable<PropertyInfo> GetBindableProperties(Type targetType)
+            => MemberAssignment.GetPropertiesIncludingInherited(targetType, _bindablePropertyFlags)
```

Comment:
I'm not super familiar with this code, but I suspect it's similar to https://github.com/aspnet/Common/blob/master/shared/Microsoft.Extensions.PropertyHelper.Sources/PropertyHelper.cs

Reply:
It is somewhat similar. I don't have any concrete reason to pull in the common implementation right now, but let me know if you think we should for some reason.